### ISSUE TITLE
Buscar información gráfica

### ISSUE THREAD

**Issue by Tulmot** [Tulmot/Sklearn-Multilabel#85]
Buscar información en otros multilabes de scikit learn para ver como dibujan los gráficos, que muestren los datos y como se dividen.

**Comment by Tulmot** [comment]
Os referís a algo de este tipo?

![imagen](https://user-images.githubusercontent.com/30071903/34442461-a6c14334-ecc2-11e7-898c-36ccab529206.png)

Esto lo he encontrado en clasificación multilabel
http://scikit-learn.org/stable/auto_examples/plot_multilabel.html

**Comment by jjrodriguez** [comment]
Podría ser como eso para el conjunto de datos.

Pero también queremos ver qué etiquetas asigna el clasificador a cada región. Ahí se ven un par de líneas (CCA o PCA) pero no muestra ningún clasificador.

Sería algo parecido a lo que hacen en http://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html

**Comment by Tulmot** [comment]
El valor que pone en cada grafica de 97,93.... es la probabilidad sobre 100? 

Y según lo que veo la idea que es hacer una comparación entre todos los algoritmos?

**Comment by jjrodriguez** [comment]
Pone que "The lower right shows the classification accuracy on the test set." Normalmente la precisión se expresa en porcentaje.

La idea inicial era hacer las representaciones gráficas para ver que es lo que hace cada método, pero ya puestos se puede hacer la comparación.
